support/modules/rootfs-image

```diff
@@ -180,6 +180,8 @@ case "$STATE" in
         fi
         if [ "$MENDER_FLASH_AVAILABLE" = 1 ]; then
             mender-flash --input-size $size --input $file --output $passive
+	elif echo "$passive" | grep "^/dev/ubi" > /dev/null; then
+	    ubiupdatevol $passive --size=$size - < $file
```

Comment:
Q: I'm guessing it cannot read directly from `$file` ?

Reply:
Hmm, actually, I didn't check. Can I leave it as is? I had to test this one manually because the test path in CI takes the mender-flash route and this is just a fallback.

Reply:
Absolutely, just got me curious when I read the man page :)

If it works I'm happy :raised_hands:

Reply:
Changed to `$file` and confirmed working! 🙂